src/components/generic/Select/Select.jsx

```diff
@@ -29,9 +33,12 @@ export default function Select({
   tooltip,
   ariaLabel,
   tabIndex,
+  error,
+  label: controlLabel,
+  required,
 }) {
   const [expanded, setExpanded] = useState(undefined);
-  const [cssClasses, sc] = webexComponentClasses('select', className, {disabled});
+  const [cssClasses, sc] = webexComponentClasses('select-container', className, {disabled});
```

Comment:
the first argument of webexComponentClasses is supposed to be the name of the component. Let's keep it as "select" and change sub-container names below, if needed.